1-js/11-async/01-callbacks/article.md

```diff
@@ -249,11 +249,11 @@ loadScript('1.js', function(error, script) {
 
 ![](callback-hell.svg)
 
-The "pyramid" of nested calls grows to the right with every asynchronous action. Soon it spirals out of control.
+"Hình chóp" của gọi lồng nhau tăng trưởng lên đến quyền với mỗi hành động không đồng bộ. Sớm nhất có thể, nó sẽ tăng vùn vụt tới mức không kiểm soát được.
```

Comment:
The "pyramid" of nested calls grows to the right with every asynchronous action.

should be translate

"Hình chóp" của việc gọi lồng nhau sẽ phình to sang bên phải ở mỗi lần gọi bất đồng bộ.